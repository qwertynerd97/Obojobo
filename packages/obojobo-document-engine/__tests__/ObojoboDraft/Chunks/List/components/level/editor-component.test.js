import React from 'react'
import renderer from 'react-test-renderer'

import Level from 'ObojoboDraft/Chunks/List/components/level/editor-component'

describe('List editor', () => {
	test('Level component', () => {
		const component = renderer.create(
			<Level
				node={{
					data: {
						get: () => {
							return { bulletStyle: 'square', type: 'unordered' }
						}
					}
				}}
			/>
		)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})

	test('Level component ordered', () => {
		const component = renderer.create(
			<Level
				node={{
					data: {
						get: () => {
							return { bulletStyle: 'alpha', type: 'ordered' }
						}
					}
				}}
			/>
		)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
