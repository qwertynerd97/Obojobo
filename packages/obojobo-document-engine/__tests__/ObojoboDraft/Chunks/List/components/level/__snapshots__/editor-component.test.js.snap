// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List editor Level component 1`] = `
<div>
  <ul
    style={
      Object {
        "listStyleType": "square",
      }
    }
  />
</div>
`;

exports[`List editor Level component ordered 1`] = `
<div>
  <ol
    style={
      Object {
        "listStyleType": "alpha",
      }
    }
  />
</div>
`;
